src/main/java/com/bloomtechlabs/fp/entities/EmploymentHistory.java

```diff
@@ -0,0 +1,61 @@
+package com.bloomtechlabs.fp.entities;
+
+import javax.persistence.*;
+
+@Entity
+@Table(name ="employment_histories")
+public class EmploymentHistory {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(name = "client_id")
+    private Long clientId;
+
+    @Column(name = "currently_employed")
+    private Boolean currentlyEmployed;
+
+    @Column(name = "skill_certifications")
+    private String skillCertifications;
+
+    public EmploymentHistory() {
+    }
+
+    public EmploymentHistory(Long clientId, Boolean currentlyEmployed, String skillCertifications) {
+        this.clientId            = clientId;
+        this.currentlyEmployed   = currentlyEmployed;
+        this.skillCertifications = skillCertifications;
+    }
+
+    public Long getId() {
+        return id;
+    }
+
+    public void setId(Long id) {
```

Comment:
This setter can change the primary key auto generated by JPA.

Reply:
looks like I had an oversite generating getters and setters. I'll fix this.